action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
     description: 'The location of the opta config file. Defaults to "opta.yml" in the root dir'
     required: false
     default: 'opta.yml'
+  api_key:
```

Comment:
wait why should github action need this. If the key is provided the first time the env is created, then it should not be needed when deploying a service

Reply:
It currently needs to be present every time-- would you like it to be stored?